config/default/meta.yml.j2

```diff
@@ -11,15 +11,16 @@ on:
   workflow_dispatch:
 
 jobs:
-  qa:
-    uses: plone/meta/.github/workflows/qa.yml@main
-  test:
-    uses: plone/meta/.github/workflows/test.yml@main
-  coverage:
-    uses: plone/meta/.github/workflows/coverage.yml@main
-  dependencies:
-    uses: plone/meta/.github/workflows/dependencies.yml@main
-  release-ready:
-    uses: plone/meta/.github/workflows/release_ready.yml@main
-  circular:
-    uses: plone/meta/.github/workflows/circular.yml@main
+{% for job_name in jobs %}
+  %(job_name)s:
+    uses: plone/meta/.github/workflows/%( job_name | replace("-", "_") )s.yml@%( branch )s
```

Comment:
Simplify the interpolation and `ref` suggestion:

```suggestion
    uses: plone/meta/.github/workflows/%(job_name)s.yml@%(ref)s
```

Oh 💡 now I see why you were doing the interpolation, for the line above... I would say we can live with an underscore there, maybe not so pretty, but it's CI, so who cares 😅 we care much more about the green tick that should appear next to it 😆